src/main/java/isaac/bastion/manager/BastionBlockManager.java

```diff
@@ -143,10 +144,22 @@ public boolean onCooldown(UUID player, BastionType type){
 		}
 		
 		for(Block block: result) {
-			toReturn.addAll(getBlockingBastions(block.getLocation(),accessors));
+			preblocking.addAll(getBlockingBastions(block.getLocation(),accessors));
 		}
-		
-		return toReturn;
+
```

Comment:
I did not do this initially because I was worried other plugins are using this method, and changing the behavior without changing the documentation is not good. Even now I'm unsure if this is a good change, but I have never done a full survey to ensure that at the least, none of the civ-plugins will be impacted by the change.

Something to consider.